ansible_collections/arista/avd/molecule/eos_validate_state/intended/test_catalogs/dc2-leaf3a.arista.com-catalog.yml

```diff
@@ -73,6 +73,13 @@ anta.tests.connectivity:
       port: Ethernet15
     result_overwrite:
       custom_field: 'Local: Ethernet15 - Remote: dc2-leaf2b Ethernet2'
+- VerifyLLDPNeighbors:
+    neighbors:
+    - neighbor_device: dc2-leaf3b.arista.com
+      neighbor_port: Ethernet16
+      port: Ethernet16
+    result_overwrite:
+      custom_field: 'Local: Ethernet16 - Remote: dc2-leaf3b.arista.com Ethernet16'
```

Comment:
hehe given that this is a remote interface we control and it is shut down on leaf3b this test will always fail.. not sure if we should think (not in this PR ;) ) to tackle this later (means need access to our peer structured_config) to detect this. @carl-baillargeon for info

Reply:
@gmuloc Yes good point. I will open an issue to track this.